source/installation/install-on-an-unattended-server/windows-server-installation/install-windows-server.txt

```diff
@@ -1,3 +1,83 @@
+.. _install-windows-server:
+
 ==================
 Install on Windows
-==================
\ No newline at end of file
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Install Relational Migrator to run as an unattended server on Windows. By running 
+as an unattended server, Relational Migrator binds to any IP and port on the server 
+and is exposes it as a web application.
+
+Steps
+-----
+
+1. Log in to the server.
+
+   You can log into the server interactively or using Remote Desktop Protocol (RDP).
+   For more information on logging into the server using Remote Desktop Protocol (RDP),
+   see `Connect to your Windows Instances using RDP <https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/connecting_to_windows_instance.html#connect-rdp>`__. 
+
+#. Download the Relational Migrator ``.msi`` installer. 
+
+   Download the Relational Migrator ``.msi`` installer to the target machine or 
+   remotely copy it over to the instance. Find the latest Relational Migrator 
+   installer on the `release page <https://migrator-installer-repository.s3.ap-southeast-2.amazonaws.com/index.html>`__.
+
+#. Launch the installer and follow the single machine :ref:`Install on Windows <single-windows>` instructions. 
```

Comment:
(suggestion)
Instead of "single machine Install on Windows" 
Consider " single machine **installation instructions**. 

The bolded text is the link text.

Reply:
Done.